pono.cpp

```diff
@@ -64,7 +61,9 @@ ProverResult check_prop(PonoOptions pono_options,
   Engine eng = pono_options.engine_;
 
   std::shared_ptr<Prover> prover;
-  if (pono_options.ceg_prophecy_arrays_) {
+  if (pono_options.cegp_abs_vals_) {
```

Comment:
we should check if the right options are set.
e.g. ceg-proph is set to true.

Reply:
I check in the `make_cegar_values_prover` function. Do you think we should check here also?

Reply:
then it is fine.